helpers.sh

```diff
@@ -282,21 +286,34 @@ git_log() {
   echo "${output/\"/\\\"}" # Escape double quotes
 }
 
+
 export_coveralls_data() {
-  local service_name
+  local service_name=""
```

Comment:
If it is an unknown environment (e.g. local) then we just leave the metadata empty.

Reply:
Maybe better to actually use `local service_name="unknown"`.

Reply:
Same with the below (`branch_name`, etc)

Reply:
changed